hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -528,6 +529,15 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 				return ctrl.Result{}, fmt.Errorf("failed to get AgentMachineTemplate: %w", err)
 			}
 		}
+	case hyperv1.KubevirtPlatform:
+		machineTemplate, err = r.reconcileKubevirtMachineTemplate(ctx, hcluster, nodePool, infraID, controlPlaneNamespace)
+		if err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to reconcile KubevirtMachineTemplate: %w", err)
+		}
+		span.AddEvent("reconciled kubevirtmachinetemplate", trace.WithAttributes(attribute.String("name", machineTemplate.GetName())))
+	case hyperv1.NonePlatform:
```

Comment:
I think this is a rebase leftover. the `None` platform is handled in `isAutomatedMachineManagement`.

Reply:
You right, removed it